CHANGELOG.md

```diff
@@ -18,7 +18,13 @@ _None_
 
 ### Internal Changes
 
-_None_
+* Updated the following internal dependencies: [#341]
+	* buildkit (1.4.5) -> (1.5.0)
+	* nokogiri (1.12.5) -> (1.13.3)
+	* oj (3.13.9) -> (3.13.11)
+	* rake-compiler (1.1.1) -> (1.1.9)
+* Updated the following public dependency: [#341]
+	* buildkit (1.4.5) -> (1.5.0)
```

Comment:
I was surprised by the indentation here, which uses tabs. But, this is also the first occurrence of an indented list, so I guess that's okay as the standard for the file 🤷

Reply:
tbh I'm not sure this was worth it to list the details of the dependencies being updated directly in the CHANGELOG 🤷 . The CHANGELOG already references `[#341]` (aka this PR) so if anyone is interested in knowing the details of what dependencies were changed/updated, they can just head here in this PR and look at the `Gemfile.lock` in the diff 🙃

Reply:
Yeah, fair – I just went over over-explaining – no need for it in the future 🤷‍♂️